services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/BatchGetResultPage.java

```diff
@@ -64,11 +85,23 @@ public static Builder builder() {
         private Builder() {
         }
 
+        /**
+         * Adds a response to the result object. Required.
+         *
+         * @param batchGetItemResponse
+         * @return a builder of this type
+         */
         public Builder batchGetItemResponse(BatchGetItemResponse batchGetItemResponse) {
             this.batchGetItemResponse = batchGetItemResponse;
             return this;
         }
 
+        /**
+         * Adds a mapper extension.
```

Comment:
Maybe we should explain why this is here. Not obvious to the uninitiated. I'm not particularly comfortable we have to expose this at all, it's really a nuance of the inner workings of how this is working behind the scenes.

Reply:
True, added context and reference.